MAUI/DataGrid/Clipboard-Operations.md

```diff
@@ -0,0 +1,546 @@
+{% endtabs %}
+
+### PasteContent
+
+PasteContent event occurs when paste the clipboard value into SfDataGrid.DataGridCopyPasteEventArgs provides information for `PasteContent` event. You can cancel paste operation by handling this event.
+
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.PasteContent += dataGrid_DataGridPasteContent;
+
+void dataGrid_DataGridPasteContent(object sender, DataGridCopyPasteEventArgs e)
+{
+    if (((sender as SfDataGrid).SelectedRow as OrderInfo).OrderID == 1010)
+    {
+        e.Handled = true;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### CopyCellContent
+
+CopyCellContent event occurs when cell being copy/cut. DataGridCopyPasteCellEventArgs provides information for `CopyCellContent` event, which has following members,
+
+* ClipBoardValue – Returns cell value.
+
+* Column – Returns corresponding DataGridColumn of a cell.
+
+* RowData – Returns corresponding RowData of a cell.
+
+You can change the text copied to clipboard by changing the `ClipBoardValue`.
+ 
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.CopyCellContent += dataGrid_DataGridCopyCellContent;
+
+void dataGrid_DataGridCopyCellContent(object sender, DataGridCopyPasteCellEventArgs e)
+{
+}
+{% endhighlight %}
+{% endtabs %}
+
+The below code example change the clipboard value as 100 instead of cell value 10003 in SfDataGrid.
+
+{% tabs %}
+{% highlight c# %}
+void dataGrid_DataGridCopyCellContent(object sender, DataGridCopyPasteCellEventArgs e)
+{
+    if (e.Column.MappingName == "OrderID" && (e.RowData as OrderInfo).OrderID == 10003)
+        e.ClipBoardValue = 100;
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Copy to Clipboard based on Cell Value in Maui DataGrid" src="Images/clipboard-operations/maui-datagrid-copy-cell-value.png" width="404"/>
+
+The below code example handled the copy operation when `MappingName` of a Column is Country.
+
+{% tabs %}
+{% highlight c# %}
+void dataGrid_DataGridCopyCellContent(object sender, DataGridCopyPasteCellEventArgs e)
+{
+    if (e.Column.MappingName == "Country")
+        e.Handled = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Copy to Clipboard based on Mapping Name in Maui DataGrid" src="Images/clipboard-operations/maui-datagrid-copy-mapping-name.png" width="404"/>
+
+### PasteCellContent
+
+PasteCellContent provides information for `PasteCellContent` event, which has following members.
+
+* ClipBoardValue – Returns clipboard value of a particular cell.
+
+* Column – Returns corresponding DataGridColumn of a cell.
+
+* RowData – Returns corresponding RowData of a cell.
+
+You can change the text paste to SfDataGrid by changing the `ClipBoardValue`.
+
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.PasteCellContent += dataGrid_PasteCellContent;
+
+void dataGrid_DataGridPasteCellContent(object sender, DataGridCopyPasteCellEventArgs e)
+{
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+The below code example change the clipboard value as Test instead of clipboard value BOLID.
```

Comment:
The example below changes the clipboard value to "Test" instead of the clipboard value "BOLID":